requirements/updater.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+set -ue
+
+export LC_ALL=C
+requirements_in="$(readlink -f ./requirements.in)"
+requirements_ansible_in="$(readlink -f ./requirements_ansible.in)"
+requirements="$(readlink -f ./requirements.txt)"
+requirements_ansible="$(readlink -f ./requirements_ansible.txt)"
+pip_compile="pip-compile --no-header --quiet -r --allow-unsafe"
```

Comment:
Do we really need `--quiet`? It might be better to allow the output through to help with debugging. I don't think it should be a problem if the script is noisy. People should only be running it when making intentional requirement changes.

Reply:
I found it hard to navigate in the output since pip-compile gets triggered multiple times. The output was exclusively what is being written into the output file if I'm not mistaken so I figured I could just make this a little bit more quiet. (I kind of prefer quiet over super noisy in builds, because if you are having too much output warnings/error messages are getting easily eaten by the sheer amount of regular output, but I understand that I'm maybe a little bit alone with this)